packages/istio/charts/1.6.500/templates/base-config-map.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-istio-operator-cr
```

Comment:
This will overwrite every time the chart is upgraded. I suppose thats ok, but we might want to change later. I might change the name to be more installation focused, but its fine.